OpenRA.Mods.Common/Widgets/Logic/ServerCreationLogic.cs

```diff
@@ -90,6 +90,10 @@ public ServerCreationLogic(Widget widget, ModData modData, Action onExit, Action
 			checkboxUPnP.OnClick = () => allowPortForward ^= true;
 			checkboxUPnP.IsDisabled = () => !Game.Settings.Server.AllowPortForward;
 
+			var labelUPnP = panel.GetOrNull<LabelWidget>("UPNP_NOTICE");
+			if (labelUPnP != null)
+				labelUPnP.IsVisible = () => !Game.Settings.Server.DiscoverNatDevices;
```

Comment:
I already removed it in https://github.com/OpenRA/OpenRA/pull/11286/files#diff-0871ba7519d8b084807b23cec86099d8 Forgot about that.